### ISSUE TITLE
The LICENSE looks outdated

### ISSUE THREAD

**Issue by janbrab** [tohecz/ctuthesis#26]
Since this template is officialy recommended by CTU FEE I believe that the LICENSE file is outdated. My main concern is the following:

> The only allowed usage of the files is for testing purposes by members of the academic community of CTU in Prague.

**Comment by jkulhanek** [comment]
I agree. This is the official template and many CTU works are written using the template. Unfortunately, all these works break the license.
